content/content.go

```diff
@@ -13,7 +13,7 @@ import (
 var (
 	errNotFound = errors.New("content: not found")
 
-	bufPool = sync.Pool{
+	BufPool = sync.Pool{
```

Comment:
This is starting to get replicated in many places and we were avoiding exporting them in the packages. Can we either use a central pool or just have the packages create their own?